ExternalModule.php

```diff
@@ -321,8 +321,18 @@ function hook_every_page_top($project_id) {
             send_rx_save_record_field($project_id, $event_id, $record, 'send_rx_dag_id', $group_id);
         }
 
+        global $Proj;
+        $bypass = array_combine($Proj->eventsForms[$event_id], $Proj->eventsForms[$event_id]);
```

Comment:
I don't know why you repeat "$Proj->eventsForms[$event_id]", the but code works perfectly.  I love the search for our field name pattern.  It's simple and very effective.